src/types.jl

```diff
@@ -248,12 +248,18 @@ for prefix in [:mag, :arf, :arb, :acb]
         Base.convert(::Type{Ptr{$TStruct}}, x::$TRef) = cstruct(x)
         Base.cconvert(::Type{Ref{$TStruct}}, x::$TRef) = cstruct(x)
 
+        _nonreftype(::Type{<:Union{$T,$TRef}}) = $T
+
```

Comment:
Assuming my comment about removing ref types above is accurate I guess this is no longer used? Could possibly keep it anyway for future use?

Reply:
yes, this is not used in the code; I'll remove it